versioning/create-release.py

```diff
@@ -129,6 +142,11 @@ def main():
     parser.add_argument("-n", action='store_false', help = "don't fetch tags")
     args = parser.parse_args()
 
+    if os.getenv('GITHUB_TOKEN') == None:
```

Comment:
I think it would be nice having this described in the help string.

Reply:
ok